byterun/Makefile.common

```diff
@@ -93,8 +93,8 @@ install-runtimed:
 
 ifeq "$(RUNTIMEI)" "true"
 install::
-	cp ocamlruni$(EXE) $(INSTALL_BINDIR)/ocamlruni$(EXE)
-	cp libcamlruni.$(A) $(INSTALL_LIBDIR)/libcamlruni.$(A)
+	cp ocamlruni$(EXE) "$(INSTALL_BINDIR)/ocamlruni$(EXE)"
```

Comment:
You're using a different mechanism for quoting `INSTALL_BINDIR` here and in `Makefile`. That can't be right.